odoo/tools/misc.py

```diff
@@ -5,53 +5,48 @@
 """
 Miscellaneous tools used by OpenERP.
 """
-from functools import wraps
-import babel
-import babel.dates
-from contextlib import contextmanager
+import cProfile
+import collections
 import datetime
-import math
-import subprocess
 import io
 import os
-
-import collections
-import passlib.utils
 import pickle as pickle_
-import pytz
 import re
 import socket
+import subprocess
 import sys
 import threading
 import time
+import traceback
 import types
 import unicodedata
-import werkzeug.utils
 import zipfile
-from collections import defaultdict, Iterable, Mapping, MutableMapping, MutableSet, OrderedDict
+from collections import defaultdict, OrderedDict
+from collections.abc import Iterable, Mapping, MutableMapping, MutableSet
```

Comment:
Maybe update only this bit? In 13.3 I updated all the imports but that seem risky / unnecessary here.